tests/interfaces/events-query.spec.ts

```diff
@@ -43,51 +42,37 @@ describe('EventsQuery Message', () => {
       expect((message.descriptor.filters![0] as ProtocolsQueryFilter).protocol).to.eq('http://example.com');
     });
 
-    it('should auto-normalize schema URL', async () => {
-      const alice = await TestDataGenerator.generatePersona();
-
-      const options = {
-        recipient : alice.did,
-        signer    : Jws.createSigner(alice),
-        filters   : [{ schema: 'example.com/' }],
-      };
-      const eventsQuery = await EventsQuery.create(options);
-
-      const message = eventsQuery.message as EventsQueryMessage;
-
-      expect(message.descriptor.filters?.length).to.equal(1);
-      expect((message.descriptor.filters![0] as RecordsFilter).schema).to.eq('http://example.com');
-    });
-
     it('allows query with no filters', async () => {
       const alice = await TestDataGenerator.generatePersona();
       const currentTime = Time.getCurrentTimestamp();
-      const eventsQueryPromise = await EventsQuery.create({
+      const eventsQuery = await EventsQuery.create({
         messageTimestamp : currentTime,
         signer           : Jws.createSigner(alice),
       });
-      expect(eventsQueryPromise.message.descriptor.filters).to.be.undefined;
+
+      expect(eventsQuery.message.descriptor.filters).to.deep.equal([]); // empty array
     });
 
     it('removes empty filters', async () => {
       const alice = await TestDataGenerator.generatePersona();
       const currentTime = Time.getCurrentTimestamp();
 
       // single empty filter fails
-      const eventsQueryPromise = EventsQuery.create({
-        filters          : [{}],
+      const eventsQuery1 = await EventsQuery.create({
         messageTimestamp : currentTime,
         signer           : Jws.createSigner(alice),
+        filters          : [{}],
       });
-      await expect(eventsQueryPromise).to.eventually.be.rejectedWith('fewer than 1 items');
+      expect(eventsQuery1.message.descriptor.filters).to.deep.equal([]); // empty array
```

Comment:
Unrelated to this PR, but didn't like the duality between an undefined filter and empty array, so made filters required and allowed an empty array in the message descriptor.